src/Microsoft.Extensions.Logging.Abstractions/EventId.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging
+{
+    public struct EventId
+    {
+        private int _id;
+        private string _name;
+
+        public EventId(int id, string name = null)
+        {
+            _id = id;
+            _name = name;
+        }
+
+        public static implicit operator int(EventId e)
```

Comment:
Do these converters need to be bidirectional? Isn't this just to help you call logger.Write(12,...)? When do you need to convert from EventId to int?